test/fixtures/test-runner/output/spec_reporter_cli.snapshot

```diff
@@ -180,14 +178,16 @@
       *
 
  callback called twice (*ms)
-  Error: callback invoked multiple times
-      *
-      *
+  'callback invoked multiple times'
 
  callback called twice in different ticks (*ms)
  callback called twice in future tick (*ms)
-  Error: callback invoked multiple times
-      *
+  Error [ERR_TEST_FAILURE]: callback invoked multiple times
```

Comment:
Is there a way to preserve the old behavior here? If I recall correctly, `ERR_TEST_FAILURE` had some special handling before.

Reply:
this it now the same as running without --test